src/views/Tickets.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { fetchTickets } from '../services/tickets';
+
+export default function Tickets() {
+  const params = useParams();
```

Comment:
We won't need to grab the params since we are not accessing the /:id of any route. We can also remove it in our useEffect dependency.

Reply:
okay thank you! So the params is only if we are calling the data by id?  for example when I create a fetchObjetcById?  Idk why I saw params passed as a dependecy without id in an assignment.  maybe I remembered it incorrectly.